examples/server/server.cpp

```diff
@@ -342,6 +342,11 @@ struct server_task {
             }
         }
 
+        if (params.sampling.n_probs > 0 && params.cache_prompt) {
+            SRV_WRN("cache_prompt is not compatible with n_probs > 0 (current value = %d), disabling cache_prompt.\n", params.sampling.n_probs);
```

Comment:
Hm, what is the reason to not be compatible?

Reply:
Hmm after second thought I think my logic here is incorrect, I'll revert it. Thanks for asking.

Indeed, I was thinking about the case where generated tokens are being cached (but we don't store their logprobs in cache). But in this case, generated tokens from last request will be used as prompt tokens for next request, so we don't even need their logprobs anyway!

Reply:
Remove this check in https://github.com/ggerganov/llama.cpp/pull/10783/commits/01afafef93ad32b8be48987a0b86649bf176e39f